cmd/config.go

```diff
@@ -214,12 +223,45 @@ var configFeatureSwitch = &cobra.Command{
 var configGetCurrent = &cobra.Command{
 	Use:     "current",
 	Aliases: []string{"cur"},
-	Short:   "Display the current lagoon that commands would be executed against",
+	Short:   "Display the current Lagoon that commands would be executed against",
 	Run: func(cmd *cobra.Command, args []string) {
 		fmt.Println(viper.GetString("current"))
 	},
 }
 
+var configLagoonVersionCmd = &cobra.Command{
+	Use:     "lagoon-version",
+	Aliases: []string{"l"},
+	Hidden:  false,
+	Short:   "Checks the current Lagoon for its version and sets it in the config file",
+	PreRunE: func(_ *cobra.Command, _ []string) error {
+		return validateTokenE(cmdLagoon)
+	},
+	RunE: func(cmd *cobra.Command, args []string) error {
+		debug, err := cmd.Flags().GetBool("debug")
+		if err != nil {
+			return err
+		}
+		current := viper.GetString("current")
+		lc := client.New(
+			viper.GetString("lagoons."+current+".graphql"),
+			viper.GetString("lagoons."+current+".token"),
+			viper.GetString("lagoons."+current+".version"),
+			lagoonCLIVersion,
+			debug)
+		lagoonVersion, err := helpers.GetLagoonAPIVersion(lc)
```

Comment:
This is another function that only seems to be used in `cmd`.. does it need to be in a separate package?